src/gatsby/src/pages/active-advisories.js

```diff
@@ -282,12 +282,30 @@ const PublicActiveAdvisoriesPage = ({ data }) => {
       }
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [pageIndex, apiCall, apiUrl]
+    [apiCall, apiUrl]
   )
 
-  // Page setter exposed to AdvisortyPageNav
-  const setPage = p => {
-    setPageIndex(p)
+  // Load more advisories when 'Load more' button is clicked
+  const handleLoadMore = () => {
+    const newIndex = pageIndex + 1
+    setPageIndex(newIndex)
+    const pageStart = (newIndex - 1) * pageLen
+
+    const aType = getAdvisoryTypeFromUrl()
+    let q = getApiQuery(aType)
+
+    let newApiCall = apiUrl + `/public-advisories` + q
+    newApiCall += "&limit=" + pageLen
+    newApiCall += "&start=" + pageStart
+    axios.get(newApiCall).then(resultResponse => {
```

Comment:
This came up in another PR recently, so I'll mention it here too: You might find the URLSearchParams API a bit easier to work with in the future https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams compared to manually building strings and managing the "&" characters 👍

Reply:
Thanks! I didn't know this but it looks very useful 😄 I'd appreciate it if you could take a second look 🙏

Reply:
Looks good! There's also the URL API for the main part of the URL, but it's not _as_ useful unless you need to parse out or change a part of an existing url: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL 🎓